### PR TITLE
Runner\Output - informations about input/output tables

### PR DESCRIPTION
**Pull request by ErikZigo**
Fixes: https://keboola.atlassian.net/browse/PS-2640

### CODE REVIEW

**Comment by ErikZigo** [comment]
Nejako mi popadaly testy:-/

**Comment by ErikZigo** [comment]
Ty jo zase ne :D

File: src/Docker/Runner/DataLoader/NullDataLoader.php [hunk 1]
```diff
@@ -25,7 +26,7 @@ public function loadInputData(
         InputTableStateList $inputTableStateList,
         InputFileStateList $inputFileStateList
     ) {
-        return new StorageState(new InputTableStateList([]), new InputFileStateList([]));
+        return new StorageState(new Result(), new InputFileStateList([]));
```
Comment (ErikZigo) [inline src/Docker/Runner/DataLoader/NullDataLoader.php:29]: 👍

**Review by odinuv — APPROVED** [review]
nezapomenout updatnout composer

**Pull request merged**